modules_HW4/fastq_filter_m.py

```diff
@@ -1,20 +1,56 @@
-def mean(array: list[int | float]) -> float:
-    return sum(array)/len(array)
-
-
-def phread_score(field_4: str) -> float:  # Функция для расчета phread score
-    symbol_collector = []
-    for symbol in field_4:
-        symbol_collector.append(ord(symbol) - 33)
-    return mean(symbol_collector)
-
-
-def check_limits(gc_bounds: tuple | int, length_bounds: tuple | int) -> tuple:
-    """Определение лимитов,
-    в том случае если подали
-    одним числом"""
-    if type(gc_bounds) is int:
-        gc_bounds = (0, gc_bounds)
-    elif type(length_bounds) is int:
-        length_bounds = (0, length_bounds)
-    return gc_bounds, length_bounds
+# import modules
+from os import mkdir
+from os.path import isdir
+from os.path import join as path_join
+from BioSEQ import filter_fastq
+
+
+def phread_score(qual: str) -> float:  # estimate the read quality
+    return sum(ord(q) - 33 for q in qual) / len(qual)
+
+
+def make_bounds(limit: float | int | tuple) -> tuple:
+    """Define limits, if it
+    specified with one case"""
+    if isinstance(limit, (int, float)):
+        return (0, limit)
+    return limit
+
+
+def read_to_filter_fastq(fastq_file) -> dict:
+    """
+    Transform 4 readline to dict
+    for filter_fastq
+    """
+    seq_read = fastq_file.readline
+    seq_dict = {
+            seq_read().strip(): (seq_read().strip(), seq_read().strip(),
+                                 seq_read().strip())[::2]
+                }
+    return seq_dict
+
+
+# Function for filter fastq file format
+def Record_filt_fastq(
+        fastq_input: str,
+        fastq_ouput: str,
+        gc_bounds=(0, 100),
+        len_bounds=(0, 2**32),
+        quality_threshold=0) -> None:
+    """
+    Function for record filt fastq_file, for input takes file fastq format
+    for output return file with same format, with filtered seqs
+    """
+    if isdir('filtered'):
```

Comment:
```suggestion
    if not isdir('filtered'):
            mkdir('filtered')
```
И далее по тексту. Вышла странная структура функции, нет никакой необходимости перевызова.